charts/weave-gitops/templates/wego-app.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wego-app
+  namespace: {{.Values.namespace}}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: wego-app
+    spec:
+      serviceAccountName: wego-app-service-account
+      containers:
+        - name: wego-app
+          image: "ghcr.io/weaveworks/wego-app:{{.Chart.AppVersion}}"
+          args: ["ui", "run", "-l", "--helm-repo-namespace","{{.Values.namespace}}"]
+          ports:
+            - containerPort: 9001
+              protocol: TCP
+          imagePullPolicy: IfNotPresent
+  selector:
+    matchLabels:
+      app: wego-app
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
Should we use a ClusterRoleBinding ? When installing the chart we have currently in the logs :
```
E0209 15:49:45.385748       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.22.2/tools/cache/reflector.go:167: Failed to watch *v1beta1.HelmRepository: failed to list *v1beta1.HelmRepository: helmrepositories.source.toolkit.fluxcd.io is forbidden: User "system:serviceaccount:flux-system:wego-app-service-account" cannot list resource "helmrepositories" in API group "source.toolkit.fluxcd.io" at the cluster scope
```

Reply:
With a clusterrolebinding it is working.
I made a PR for change suggestions https://github.com/weaveworks/weave-gitops/pull/1422

Reply:
@mtparet This a proof of concept idea but I did put up this draft pr and ask team members why I was getting this error. So thank you for helping out with that. Most likely the chart will not actually sit in this repo.